docs/release-notes/cloud/cloud-2023-09.md

```diff
@@ -0,0 +1,96 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Cloud September 2023.
+included_jira_versions: ['2023.09']
+codacy_tools_version_old: https://github.com/codacy/codacy-tools/releases/tag/7.8.2
+codacy_tools_version_new: https://github.com/codacy/codacy-tools/releases/tag/7.10.32
+---
+
+# Cloud September 2023
+
+These release notes are for the Codacy Cloud updates during September 2023.
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+<!--TODO Check these issues manually
+
+Jira issues with disabled release notes
+
+Epics:
+-   https://codacy.atlassian.net/browse/ALA-573
+-->
+
+## Product enhancements
+
+-   It's now possible to send notifications to a Slack channel of your choice whenever Codacy detects new critical Security issues in the default branch of any repository in your organization. See [how to install the Codacy Slack integration for Security issues](../../organizations/integrations/slack-integration.md). (TAROT-2242)
+
+    ![Slack integration for Security issues](../images/tarot-2242.png)
+
+-   You can now use the new [Codacy Visual Studio Code extension](https://github.com/codacy/codacy-vscode-extension) to get the full list of problems found by Codacy for a pull request and navigate to any Quality issue that you want to review and fix. See [how to install and use the Codacy Visual Studio Code extension](../../getting-started/integrating-codacy-with-visual-studio-code.md). (ALA-549)
```

Comment:
Feels like we are missing the selling point here. 
"to get the full list of problems found by Codacy for a pull request" -codacy does that, with the plugin we are bringing that to the IDE.
"and navigate to any Quality issue that you want to review and fix" - similar.

@kendrickcurtis @mrfyda thoughts?

Reply:
I think that is what the sentence is trying to convey ☝️ 

maybe what we have on the marketplace listing is more clear:
> The Codacy extension for Visual Studio Code helps you review and fix issues found by Codacy by highlighting problematic code patterns, notifying you whether a pull request is up to standards, and displaying code quality metrics directly within Visual Studio Code.

Reply:
@RaquelHipolito @mrfyda Thank you for the input! I tweaked the sentence, let me know if it's ok now.